core/tests/data/dummy_ci_tests/dummy_protractor.conf.js

```diff
@@ -0,0 +1,21 @@
+var suites = {
+     accessibility: [
+      'protractor/accessibility.js'
+    ],
+
+    additionalEditorFeatures: [
+      'protractor_desktop/additionalEditorFeatures.js'
+    ],
+
+    additionalPlayerFeatures: [
+      'protractor_desktop/additionalPlayerFeatures.js'
+    ],
+
+    topicsAndSkillsDashboard: [
+      'protractor_desktop/topicsAndSkillsDashboard.js'
+    ],
+    
+    users: [
+      'protractor_desktop/userJourneys.js',
+    ],
+}
```

Comment:
Missing newline at EOF. Also indentation in this file should be by 2 spaces, not 4.

Reply:
Done.